.appveyor.yml

```diff
@@ -0,0 +1,18 @@
+platform:
+  - x86
+  - x64
+
+init:
+  - git config --global core.autocrlf input

```

Comment:
I'm not sure if this is needed.
Moreover, shouldn't use ```core.autocrlf true```?

Reply:
actually it is not needed.